bc_obps/registration/tests/endpoints/test_useroperator_endpoints.py

```diff
@@ -487,6 +487,37 @@ def test_cas_admin_can_update_status_of_a_user_operator(self, mocker):
             user_operator.user.email,
         )
 
+    def test_cas_admin_can_update_status_of_a_user_operator_with_new_operator(self, mocker):
+        operator = operator_baker({'status': Operator.Statuses.APPROVED, 'is_new': False, 'created_by': self.user})
```

Comment:
If it's a new operator shouldn't `is_new` be true? Status is also probably pending?

Reply:
Sorry, copy/pasting 🤦‍♂️

Reply:
actually, this is correct 😆  because here we are testing the user operator status change and not the operator change so we have to mark the operator as `is_new=False` and `status=APPROVED` so we can bypass the below part and can get to the email sending part:

```python
# We can't update the status of a user_operator if the operator has been declined or is awaiting review, or if the operator is new
        if operator.status == Operator.Statuses.DECLINED or operator.is_new:
            raise Exception("Operator must be approved before approving or declining users.")
```

Reply:
Right! In that case, maybe we don't need two tests then?